packages/forms/src/UIForm/fields/ResourcePicker/ResourcePicker.component.js

```diff
@@ -67,8 +81,11 @@ class ResourcePicker extends Component {
 			selected.push(id);
 		}
 
+		const value = multi ? selected : selected[0];
 		this.setState({ filters: { ...this.state.filters, selected } });
-		this.onChange(event, multi ? selected : selected[0]);
+		this.onChange(event, value);
+
+		this.onTrigger(event, CHANGE, { value });
```

Comment:
Can you add a `triggers` part in the widget README please ?
- explain the 2 supported events
- a little example of how to use them (uischema, onTrigger callback)

Reply:
Please add a test for this.